docs/4.0/examples/navbar-static/index.html

```diff
@@ -45,7 +45,7 @@
     <main role="main" class="container">
       <div class="jumbotron">
         <h1>Navbar example</h1>
-        <p class="lead">This example is a quick exercise to illustrate how the top-aligned navbar works. As you scroll, this navbar remains in its original position and moves with the rest of the page.</p>
+        <p class="lead">This example is a quick exercise to illustrate how the static navbar works.</p>
```

Comment:
Why'd you change this? My approach in some of these is to use language that's not the exact class or component name.

Reply:
Well, does the text reflect the example? It was the same as the fixed navbar example...

Reply:
The original text does describe the example, yes. This does, too, but not as well. Perhaps I need to update the fixed top one?

Reply:
No, wait, I had read it wrong. My bad, I reverted this.